src/test/java/lotto/model/LottoGeneratorTest.java

```diff
@@ -1,16 +1,36 @@
 package lotto.model;
 
-import lotto.util.LottoGenerator;
+import org.assertj.core.api.Assertions;
 import org.junit.jupiter.api.DisplayName;
 import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.ValueSource;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
 public class LottoGeneratorTest {
 
-    @DisplayName("LottoGenerator를 통해 로또가 잘 생성 되는지 테스트")
+    @DisplayName("자동 로또 생성 되는지 테스트")
     @Test
     void makeLottoTest() {
-        assertThat(LottoGenerator.makeLotto().getClass()).isEqualTo(Lotto.class);
+        assertThat(LottoGenerator.makeBunchOfAutoLotto(1).getBunchOfLotto().get(0).getClass()).isEqualTo(Lotto.class);
+    }
+
+    @DisplayName("수동 로또 생성 테스트")
+    @ParameterizedTest
+    @ValueSource(strings = {"1,2,3,4,5,6"})
+    void makeManualLottoTest(String inputNumber) {
+
+        assertThat(LottoGenerator.makeManualLotto(inputNumber).getClass()).isEqualTo(Lotto.class);
```

Comment:
```suggestion
        assertThat(LottoGenerator.makeManualLotto(inputNumber)).isInstanceOf(Lotto.class);
```
`getClass()`를 통해 class 정보를 가져올 필요 없이, assertJ의 함수를 이용하면 좋을 것 같아요 👀